functions/security/main.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_bearer_token]
+import requests
+
+# TODO<developer>: set these values
+REGION = 'us-central1'
+PROJECT_ID = 'my-project'
+RECEIVING_FUNCTION = 'my-function'
```

Comment:
There is only one function in this file. Rename to `FUNCTION_NAME`?

Reply:
This `calling_function` is supposed to call a second "receiving function" with the token, wait for results (via an HTTP response), and return those results to its caller.

Reply:
This is fine. Would be nice to have a comment or a better function name rather than `my-function`.